config/locales/activerecord.yml

```diff
@@ -120,8 +120,8 @@ en:
       must_be_am_pm: Select am or pm
     starts_on:
       before_today: The date the job will be listed must be in the future
-      before_publish_on: The date the job will be listed must be after the date the job will be listed
-      before_expires_on: The date the job will be listed must be after the date applications are due
+      before_publish_on: The date the job starts must be after the date the job listing will be published
+      before_expires_on: The date the job starts must be after the date applications are due
```

Comment:
Shouldn't this be `The date the job starts must be after the closing date`?

Reply:
Bloody hell yeah. Just copied and pasted the content I received without reading. Ooops.